src/main/java/pb360/model/RestAPI.java

```diff
@@ -0,0 +1,76 @@
+package pb360.model;
+
+import java.util.Date;
+import java.util.List;
+
+public class RestAPI {
```

Comment:
add @JsonInclude(IncludeJson.non-default) please research on google to understand it also explain about this configuration on mail